inc/saiqueue.h

```diff
@@ -73,12 +73,32 @@ typedef enum _sai_queue_attr_t
     SAI_QUEUE_ATTR_TYPE = SAI_QUEUE_ATTR_START,
 
     /**
+     * @brief Pord id
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_PORT
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY | KEY
+     */
+    SAI_QUEUE_ATTR_PORT = 0x00000001,
+
+    /**
      * @brief Queue index
      *
      * @type sai_uint8_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY | KEY
      */
-    SAI_QUEUE_ATTR_INDEX = 0x00000001,
+    SAI_QUEUE_ATTR_INDEX = 0x00000002,
+
+    /**
+     * @brief Parent scheduler node
+     *
+     * In case of Hierarchical Qos not supported, the parent node is the port
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_SCHEDULER, SAI_OBJECT_TYPE_PORT
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
```

Comment:
updated. since HQoS is not supported, so let's keep the flags as MANDATORY_ON_CREATE, once the HQoS is introduced we can remove the flag.